packages/gasket-engine/package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-json": "^2.1.2",
     "eslint-plugin-mocha": "^8.0.0",
-    "jest": "^26.6.3"
+    "jest": "^27.4.3"
```

Comment:
I was investigating the failing tests and came across this as well. To pick up all packages, please update the align-packages script [here](https://github.com/jpage-godaddy/gasket/blob/type-decls/scripts/align-packages.js#L37). Then run `npm run align-packages` at the root of the repo.

Before you run tests, you also need to update this particular spec file:
`packages/gasket-redux/test/configure-make-store.spec.js`
to include the following at the top ([docs](https://jestjs.io/docs/configuration#testenvironment-string)):
```
/**
 * @jest-environment jsdom
 */
```

Reply:
Thanks, that fixed it.